eox_tenant/utils.py

```diff
@@ -40,3 +43,29 @@ def move_signupsource(old_domain, new_domain):
         count = UserSignupSource.objects.filter(site=old_domain).update(site=new_domain)
 
         log.info("Updated {} SignupSources from {} to {}.".format(count, old_domain, new_domain))
+
+
+def synchronize_tenant_organizations(instance):
```

Comment:
So far this method only adds new orgs, do we have anything to remove old orgs and purge them?